features/custom_ca_k8s_csr.md

```diff
@@ -0,0 +1,230 @@
+
+[//]: # (The documentation instructions may exist on the developer wiki or the team drive. They may include instructions)
+[//]: # (for building running a `istioctl experimental command`, or using the preview profile,)
+[//]: # (or any other relevant information.)
+
+- [ ] [Feedback plan](insert_your_link_here).
+
+[//]: # (This may include user feedback meetings, discuss.istio.io conversations, GitHub issues, or mailing lists.)
+
+- [X] Disabled by default.
+
+- [X] No impact on performance when the feature is disabled.
+
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [X] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+---
+
+## Alpha
+
+### Requirements:
+
+**Design**
+
+- [] RFC has been approved describing the intention of the feature as well as the user stories behind the feature.
+
+**Config**
+
+- [] Explicit user action is required to enable this feature (e.g. a config field, config resource, or installation action).
+
+**Docs**
+
+- [] Reference docs are published to preliminary.istio.io or the Istio wiki.
+
+- [] Basic feature docs are published on preliminary.istio.io describing what the feature does, how to use it, and any caveats. 
+
+- [] Release notes entries added as appropriate
+
+- [] Upgrade notes entries added as appropriate
+
+**Tests**
+
+- [] Automated integration tests cover core use cases with the feature enabled.
+- [] When disabled, the feature does not affect system stability or performance.
+
+**API**
+
+- [ ] Initial API review.
+
+**Approvals**
+
+- [] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [] The TOC has reviewed and approved promotion of the feature as part of the
+	roadmap for a release.
+
+**Promotion**
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+
+---
+
+## Beta
+
+### Requirements:
+
+**Design**
+
+- [] Design doc describing the intention of the feature, how it will be
+	implemented, and any thoughts on how to test the feature has been approved by
+	relevant work group leads
+- [] Feature coverage and test plans written and approved.
+
+**Docs**
+
+- [ ] Documentation on istio.io includes performance expectations; may have caveats.
+- [] Documentation on istio.io includes samples/tutorials.
+- [] Documentation on istio.io includes appropriate glossary entries.
+- [] All new documentation containing user actions includes istio.io tests.
+- [] Release notes have been added.
+- [] Upgrade notes have been added.
+
+**Tests**
+
+- [] Integration tests cover feature edge cases
+- [] End-to-end tests cover samples/tutorials
+- [] Fixed issues have tests to prevent regressions
```

Comment:
```suggestion
- [ ] Integration tests cover feature edge cases
- [ ] End-to-end tests cover samples/tutorials
- [ ] Fixed issues have tests to prevent regressions
```